packages/aws-lambda/layer/bin/publish-layer.sh

```diff
@@ -160,16 +165,13 @@ if [[ -z $AWS_ACCESS_KEY_ID ]] || [[ -z $AWS_SECRET_ACCESS_KEY ]]; then
   echo Warning: AWS_ACCESS_KEY_ID or AWS_SECRET_ACCESS_KEY are not set. This might be okay if you have set up AWS authentication via other means. If not, the AWS cli commands to publish the layer will fail.
 fi
 
-echo "step 1/9: fetching AWS regions (skipping, using fixed list of regions for now)"
-
-# Actually, this should give us the regions where the Lambda service is provided:
-# REGIONS=$(aws ssm get-parameters-by-path --path /aws/service/global-infrastructure/services/lambda/regions --output text --query "Parameters[].Value" | tr '\t' '\n')
-# But for some reason, publishing to all of these regions does not work. In particular, the
-# following regions either require special authorization/subscription status or don't support Lambdas: ap-east-1 me-south-1 ap-northeast-3
+echo "step 1/9: AWS regions"
```

Comment:
_suggestion:_ Let's move this `echo` above the `aws ssm` call (and maybe print "fetching AWS regions" again as before), to give an indication of what is currently happening.

Reply:
Sure 👍

Reply:
The problem with this suggestion is that we are fetching the regions before we print the confirm prompt.
The step printing should come after the confirm prompt. 

How about we just remove this step?

Reply:
Is there a specific reason why we don't want to print any messages before the prompt? I think it is fine to 
* print "step 1: I am fetching all the regions now", 
* then print all the env vars and settings and 
* then ask via the prompt whether it's fine to continue.

I'm mostly thinking about the CI job here anyway, where the prompt is off. If the CI job fails, I want to see immediately which step failed.

Reply:
Ok why not